apps/mockingbird/README.md

```diff
@@ -0,0 +1,56 @@
+
+#### Usage
+
+MBird.py script can be used to create MBirds on the Mockingbird host. You can use it to instantiate a number of devices of a specific type and with proprietary handle names. The script can be run locally on the MBH or externally on a PC with access to the MBH network. 
+
+After successful MBird creation, the script will generate .csv and .scr files that can be used for provisioning the devices through the CMS or in the NodeUtil tool. Note that the script will create a .csv and .scr files if they don't exist and append the lines to the existing files. To provision newly created MBirds on the UUT, just import the .csv in the CMS, select the devices that appear after importing, and click 'Provision Selected' in the Devices widget. 
+
+The script's help guide can be accessed by running the  `python mbird.py -h` or `python mbird.py --help`
+
+To create MBirds of a specific type and handle use the following command:  
+
+```shell
+python mbird.py -t <Target SmartServer IP> --hdl <Devices handle> -n <Number of devices> -x <Device type XIF id> -d <Device type> -i <LON interface>
+```
+
+- <Target SmartServer IP>  - IP of the Mockingbird host.  If not supplied, default is localhost.
+- <Devices handle> - Base name of the MBirds to be created. Every unique MBird will have the base name and the device number in its name "<name>-<nr>". If the provided device handle is separated with the "-" followed by a number (e.g. , `m-tier2-50`), the MBird creation will start from the provided number and increment `n` times, where `n` is provided in the next argument. If there is more than one dash in the string, then the incrementing occurs after the last dash. For the example with `-n 10` the device handles would be: `m-tier2-50`, `m-tier2-51`, ..., `m-tier2-60`. 
+- <Number of devices> - Number of MBirds to be created.  If not supplied, default is 1
+- <Device type XIF id> - 16 character XIF identifier for the device type. If not provided the script will use the specified <Device type> with .xif appended after removing any "-<n>" suffix.
+- <Device type> - Name of the device type used in the CMS. If not specified the script will use the provided <Device type XIF id> in the format "XIF-1". 
+- <LON interface>  - name of the LON interface on the MBH to use when creating MBirds. Example `lon0`, `lon1`...
```

Comment:
There seem to be a number of additional arguments supported.  Are these intentionally omitted?